src/components/SlateEditor/plugins/table/__tests__/combined-test.ts

```diff
@@ -19,7 +19,7 @@ const editor = withHistory(withReact(withPlugins(createEditor(), learningResourc
 describe("combined table plugin tests", () => {
   test("id in th and td is preserved on serialize and normalize", () => {
     const html =
-      '<section><table><tbody><tr><th id="r0" scope="row" align="right"><p>1</p></th><td align="right" id="abc"><p>2</p></td></tr></tbody></table></section>';
```

Comment:
Id måtte flyttes bak her for at den skal være like som i serializer. Jeg aner ikke hvordan denne testen her gikk igjennom før.